oxd-gen-client/pom.xml

```diff
@@ -1,292 +1,291 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <artifactId>oxd-gen-client</artifactId>
-    <packaging>jar</packaging>
-    <name>oxd-gen-client</name>
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
We maintain pom.xml. Here it is not formatter and it seems some plugin configurations are changed. Do all tests pass with this PR ?

Reply:
We have formatted pom.xml. All test cases are running.